src/components/UserDetail/UserDetail.js

```diff
@@ -162,15 +167,17 @@ class UserDetail extends React.Component {
       .join(',');
 
     const openRequestsPath = `/requests?query=${user.barcode}&filters=${openRequestStatuses}&sort=Request date`;
-
-    return (
-      <Link
-        data-test-open-requests-count
-        to={openRequestsPath}
-      >
-        {openRequestsCount}
-      </Link>
-    );
+    if (stripes.hasPerm('ui-checkout.viewRequests')) {
```

Comment:
it looks like a separate component

Reply:
Maybe `<IfPermission>` from `@folio/stripes/core` ?